### DISCUSSION TITLE
ChoParquetWriter

### DISCUSSION THREAD

**Discussion by trunklegs911** [Cinchoo/ChoETL#277] [General]
Is this namespace no longer supported? I have added the most recent release to my .NET project and I have access to ChoCSVWriter and others, but ChoParquetWriter gives me an error that the type or namespace cannot be found. 

Anyone else having this issue? I have found nothing in the documentation to suggest that this is so longer part of the library.

Thank you!

**Comment by Cinchoo** [comment]
Install the parquet plug-in from 

https://www.nuget.org/packages/ChoETL.Parquet
